parameters/configuring-workflow-inputs.md

```diff
@@ -0,0 +1,27 @@
+# Configuring Workflow Inputs
+
+After publishing a parameterized workflow, you can trigger new runs of that workflow with different parameters. 
+This allows you to easily re-run the same workflow with different parameters, without having to re-publish the workflow.
+
+Assuming we have publish a workflow with a string parameter named "country":
+
+```python
+churn = predict_churn(reviews, country_param)
+flow = client.publish_flow("Churn Prediction", artifacts=[churn])
+
+# Sleep for 10 seconds to wait for flow to finish.
+import time
+time.sleep(10) 
+
+# This will trigger a new run of the already published flow "Churn Prediction",
+# but with the country parameter value set to "China".
+client.trigger(flow.id(), parameters={"country": "China"})
+```
+
+{% hint style="info" %}
```

Comment:
Tiny nit — having two hints next to each other might be confusing. Let's move this before the code snippet.